sdk/core/azure-json/src/main/java/com/azure/json/JsonBoolean.java

```diff
@@ -0,0 +1,64 @@
+package com.azure.json;
+
+import java.io.IOException;
+
+/**
+ * Class representing the JSON boolean type
+ */
+public class JsonBoolean extends JsonElement {
+    /**
+     * Stores the String representation of the JsonBoolean object.
+     * Can only be either "true" or "false".
+     */
+    private String booleanValue;
+
+    /**
+     * Default constructor
+     * Default sets booleanValue to "true" through the other constructor.
+     * TODO: may need to remove this due to design guidelines? May only want to
+     * have the public JsonBoolean(boolean value) constructor.
+     */
+    public JsonBoolean() { this(true); }
```

Comment:
Usually, we won't to follow defaults of the language, which for boolean would be `false`. Similar to what `AtomicBoolean`'s empty constructor does where the internal boolean value would be false.